infrastructure/zk/src/docker.ts

```diff
@@ -43,9 +44,7 @@ async function _build(image: string, tag: string) {
     }
 
     const imageTag = `-t rsksmart/rollup-${image}:${tag}`;
-    await utils.spawn(
-        `DOCKER_BUILDKIT=1 docker build ${imageTag} --platform linux/amd64 -f ./docker/${image}/Dockerfile .`
-    );
+    await utils.spawn(`DOCKER_BUILDKIT=1 docker build ${imageTag} -f ./docker/${image}/Dockerfile .`);
```

Comment:
Why did we remove the platform argument?

Reply:
to avoid having issues while building the docker image in Mac.